core/native/src/main/scala/zio/ClockPlatformSpecific.scala

```diff
@@ -52,102 +53,53 @@ private[zio] trait ClockPlatformSpecific {
   }
 }
 
-/**
- * Copy-pasted from scala.scalanative.loop
- *
- * This is temporary until we figure out why using it directly raises errors
- */
 private object ClockPlatformSpecific {
-  import scala.collection.mutable
-  import scala.concurrent.Future
-  import scala.scalanative.annotation.alwaysinline
-  import scala.scalanative.libc.stdlib
-  import scala.scalanative.loop.EventLoop
```

Comment:
Are still using even anything from `scala-native-loop`at this point? 
If not, we could remove the dependency.

Reply:
Just removed it!